app.js

```diff
@@ -30,6 +30,7 @@ const {
 	JWT_TIMEOUT_SECONDS,
 	BACKEND_URL,
 	PUBLIC_BACKEND_URL,
+	FEATURE_NUXT_ENABLED,
```

Comment:
I do not like the name of this variable because it is tightly bound to an action that does not relate in any way to the name. I would prefer something like `FEATURE_DATASOURCES_ENABLED` and would prefer to bind this to user permissions.

Reply:
The sidebaritem should instead be shown based on the user permission `DATASOURCES_VIEW`
https://github.com/schul-cloud/nuxt-client/pull/500/files